cli/cmd/vulnerability.go

```diff
@@ -270,6 +279,27 @@ func setDetailsFlag(cmds ...*flag.FlagSet) {
 	}
 }
 
+func setFailOnSeverityFlag(cmds ...*flag.FlagSet) {
+	for _, cmd := range cmds {
+		if cmd != nil {
+			cmd.StringVar(&vulCmdState.FailOnSeverity, "fail_on_severity", "",
+				fmt.Sprintf("specify a severity threshold to fail if vulnerabilities are found (%s)",
+					strings.Join(api.ValidEventSeverities, ", ")),
+			)
+		}
+	}
+}
+
+func setFailOnFixableFlag(cmds ...*flag.FlagSet) {
+	for _, cmd := range cmds {
+		if cmd != nil {
+			cmd.BoolVar(&vulCmdState.FailOnFixable, "fail_on_fixable", false,
```

Comment:
Is `fail_on_fixable` the right name for this? It's always used in conjunction with `fail_on_severity` so is the 'fail_on' redundant.